aws/required-permissions.md

```diff
@@ -129,3 +131,82 @@ Check out the templates directly for more information.
+| `kms:Decrypt` | Action | **Optional.** Decrypt messages from encrypted `ScannerDLQ`, `PostScanActionDLQ`, s3 Bucket, or Data key for scan result encryption. |
+| `kms:GenerateDataKey` | Action | **Optional.** Generate a Data key used to encrypt the scan result messages sent to `ScanResultTopic` or the error messages sent to encrypted `ScannerDLQ` and `PostScanActionDLQ`. |
+| `lambda:CreateAlias` | Action | Create an alias for Lambda functions created by Cloud One File Storage Security. |
+| `lambda:DeleteAlias` | Action | Delete the alias of Lambda functions created by Cloud One File Storage Security. |
+| `lambda:GetFunctionConfiguration` | Action | Retrieve the configuration of Lambda functions created by Cloud One File Storage Security. |
+| `lambda:GetLayerVersion` | Action | Retrieve the information of a version of the Lambda layers created or managed by Cloud One File Storage Security. |
+| `lambda:InvokeFunction` | Action | Let AWS service `EventBridge` and `SNS` invoke the Lambda functions created by Cloud One File Storage Security. |
+| `lambda:PublishVersion` | Action | Publish a new version of the Lambda functions created by Cloud One File Storage Security. |
+| `lambda:UpdateAlias` | Action | Update the alias of Lambda functions created by Cloud One File Storage Security. |
+| `lambda:UpdateFunctionConfiguration` | Action | Update the configuration of Lambda functions created by Cloud One File Storage Security. |
+| `organizations:DescribeOrganization` | Action | Retrieve information about the organization that the current AWS account belongs to. |
+| `sns:Publish` | Action | Publish a message to a `ScanResultTopic` created by Cloud One File Storage Security. |
+| `sqs:DeleteMessage` | Action | Delete messages from the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security. |
+| `sqs:GetQueueAttributes` | Action | Get the attributes of the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security. |
+| `sqs:ReceiveMessage` | Action | Receive messages from the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security. |
+| `sqs:SendMessage` | Action | Send scan messages to the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security. |
+| `ssm:GetParameter` | Action | Retrieve values in AWS Systems Manager that Cloud One File Storage Security created. |
+| `sts:AssumeRole` | Action | Grant the permissions to different principals to switch roles created by Cloud One File Storage Security. |
+| `s3:DeleteObject` | Action | Remove the malicious object from the scanning bucket after being quarantined. |
+| `s3:GetObject` | Action | Retrieve the object to be scanned or quarantined from Amazon S3. |
+| `s3:GetObjectTagging` | Action | List the tags of the scanned object. |
+| `s3:PutObject` | Action | Put the malicious object to quarantine bucket. |
```

Comment:
```suggestion
| `s3:PutObject` | Action | Puts the malicious object in the quarantine bucket |
```
The third cell in this row contains a clause phrase, not a full sentence, so doesn't require a period. The verb form should be the same for each clause in this column. Because the implied subject (the managed policy) is singular, the verb should be singular which typically ends in "s".